### ISSUE TITLE
acdc dataset dice

### ISSUE THREAD

**Issue by Wbbbjikj** [Beckschen/TransUNet#128]
I trained the ACDC dataset using transunet, but the dice value is only 85. However, the original paper can reach 89. I don't know what the reason is.

**Comment by talebim** [comment]
> I trained the ACDC dataset using transunet, but the dice value is only 85. However, the original paper can reach 89. I don't know what the reason is.

Did you changed any hyper parameter of TransUnet?
When I run TransUnet on ACDC dataset, loss don't decrease.

**Comment by Wbbbjikj** [comment]
Yes, I tried many methods to adjust parameters, and finally found that only changing batchsize to 8 could improve the performance, but it could only stay at about 87 in the end. Did I adjust the parameters incomplete? Could you please provide your parameter Settings, thank you

**Comment by laodetangkoushui** [comment]
> > I trained the ACDC dataset using transunet, but the dice value is only 85. However, the original paper can reach 89. I don't know what the reason is.
> 
> Did you changed any hyper parameter of TransUnet? When I run TransUnet on ACDC dataset, loss don't decrease.

Hello, can you provide me with the processed acdc data set? If you can provide me with a copy of the acdc data set, I will be very grateful

**Comment by laodetangkoushui** [comment]
> Yes, I tried many methods to adjust parameters, and finally found that only changing batchsize to 8 could improve the performance, but it could only stay at about 87 in the end. Did I adjust the parameters incomplete? Could you please provide your parameter Settings, thank you
Hello, can you provide me with the processed acdc data set? If you can provide me with a copy of the acdc data set, I will be very grateful

**Comment by laodetangkoushui** [comment]
> Yes, I tried many methods to adjust parameters, and finally found that only changing batchsize to 8 could improve the performance, but it could only stay at about 87 in the end. Did I adjust the parameters incomplete? Could you please provide your parameter Settings, thank you

Hello, can you provide me with the processed acdc data set? If you can provide me with a copy of the acdc data set, I will be very grateful

**Comment by xinnvY** [comment]
> > I trained the ACDC dataset using transunet, but the dice value is only 85. However, the original paper can reach 89. I don't know what the reason is.
> 
> Did you changed any hyper parameter of TransUnet? When I run TransUnet on ACDC dataset, loss don't decrease.

Hi ! it's so excited for me to see you have completed the experiment on ACDC wonderfully!!! and i also want to reproduce it,could you share your preprocessed ACDC dataset with me? 2863583613@qq.com thanks!

**Comment by zhangxuan-thecastle** [comment]
> > 我使用 transunet 训练了 ACDC 数据集，但骰子值只有 85。但是，原始论文可以达到89。我不知道是什么原因。
> 
> 您是否更改了TransUnet的任何超参数？当我在ACDC数据集上运行TransUnet时，损失不会减少。
您好，您能为我提供处理后的 acdc 数据集吗？如果您能向我提供acdc数据集的副本，我将不胜感激。我的电子邮件地址是 [1976939023@qq.com]

**Comment by zhangxuan-thecastle** [comment]
> I trained the ACDC dataset using transunet, but the dice value is only 85. However, the original paper can reach 89. I don't know what the reason is.
      Hello, can you provide me with the processed acdc data set? If you can provide me with a copy of the acdc data set, I will be very grateful. My email address is [1976939023@qq.com]

**Comment by zhangxuan-thecastle** [comment]
> > I trained the ACDC dataset using transunet, but the dice value is only 85. However, the original paper can reach 89. I don't know what the reason is.

Hello, can you provide me with the processed acdc data set? If you can provide me with a copy of the acdc data set, I will be very grateful. My email address is [1976939023@qq.com]

**Comment by ylj183** [comment]
> 我使用transunet训练了ACDC数据集，但是dice值只有85，而原论文可以达到89，不知道是什么原因。

你好，我想请问一下你数据集是怎么划分的呀？70-10-20，一共有多少个切片呢
